stablehlo/reference/Ops.cpp

```diff
@@ -384,6 +384,16 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto rhs = scope.findTensor(complexOp.getRhs());
       auto result = evalComplexOp(lhs, rhs, complexOp.getType());
       scope.add(complexOp.getResult(), result);
+    } else if (auto compositeOp = dyn_cast<CompositeOp>(op)) {
+      auto operands = scope.findTensors(compositeOp.getOperands());
+      SmallVector<InterpreterValue> values(operands.size());
+      std::transform(operands.begin(), operands.end(), values.begin(),
+                     [](const Tensor &t) { return InterpreterValue(t); });
+      auto func =
+          llvm::cast<func::FuncOp>(op.getParentOfType<ModuleOp>().lookupSymbol(
```

Comment:
Let's use that "nearestSymbol" API here as well

Reply:
Unfortunately, this requires implementing SymbolUserOpInterface, which I'm not sure we should do. Let's merge the discussion into that thread: https://github.com/openxla/stablehlo/pull/2024#discussion_r1513563283